Sources/Actions/Admin/ACP.php

```diff
@@ -1523,8 +1523,7 @@ public static function getFileVersions(array &$versionOptions): array
 		);
 
 		$ignore_sources = [
-			Config::$sourcedir . '/minify/*',
-			Config::$sourcedir . '/ReCaptcha/*',
+			Config::$vendordir . '/*',
```

Comment:
If I'm reading the code right, this line can go because ``$vendordir` is outside of `$sourcedir`, no?